build/loaders/component-example-loader.js

```diff
@@ -3,7 +3,8 @@ const pretty = require('pretty')
 const prettier = require('prettier')
 const path = require('path')
 const compiler = require('vue-template-compiler')
-const transpile = require('vue-template-es2015-compiler')
+const { transform } = require('babel-core')
+const transpile = code => transform(code, {extends: path.join(__dirname, '../../.babelrc')}).code
```

Comment:
You can use the `resolvePath`

Reply:
fix 4a68465dcf85ffdef00cdd3936d7e764e8d78ecb